shell/probe/raindrop.ps1

```diff
@@ -33,13 +33,24 @@ while ($true) {
             "dat" = $dat
             "version" = "2.1"
         } -UseBasicParsing -MaximumRedirection 0 -ErrorAction SilentlyContinue
-        
+
         $uuid = $task.content -replace ".*?([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}).*", '$1'
         $auth = $task.content -replace '^[^/]*//([^/]*)/.*', '$1'
 
         if ($uuid -and $auth -eq $ca) {
             Invoke-WebRequest -Uri $task.content -OutFile (New-Item -path "$dir\$uuid.exe" -Force ) -UseBasicParsing
-            $code = Execute "$dir\$uuid.exe"
+            $stdoutTempFile = New-TemporaryFile
```

Comment:
Feels like `New-TemporaryFile`s should be in $dir too.

Reply:
are we confident that no matter what the user's set up, our probe will have write/read access to the temp folder? why not write these files in $dir where we know we have access?

Reply:
why do we need to write to a temp file? what's the difference between that and just reading stdout/err from the process obj we get back? https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.process.standarderror?view=net-8.0#system-diagnostics-process-standarderror

Reply:
for whatever reason `Start-Process` decided to make it very difficult to do so. There are proposed ways to do it, but they risk deadlock under certain conditions:
* https://stackoverflow.com/questions/8761888/capturing-standard-out-and-error-with-start-process
* https://www.reddit.com/r/PowerShell/comments/neqw5a/with_the_startprocess_command_is_there_any_way_to/

redirecting to the file is more foolproof. Agreed on writing to $dir.